Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -70,6 +83,18 @@ class HistoryPanel: UIViewController, HomePanel {
         tableViewController.didMoveToParentViewController(self)
     }
 
+    override func viewWillAppear(animated: Bool) {
+        super.viewWillAppear(animated)
+
+        if let recentlyClosedTabs = profile.prefs.dictionaryForKey("recentlyClosedTabs") where recentlyClosedTabs["urls"]!.count > 0 {
+            recentTabsButton.enabled = true
+            recentTabsButton.alpha = 1
+        } else {
+            recentTabsButton.enabled = false
+            recentTabsButton.alpha = 0.5
```

Comment:
Nit: since the `alpha` is a visual side effect of the `enabled` state, I'd move the `alpha` parts into a `didSet` observer on `enabled` in `RecentTabsButton`.